### PR TITLE
fix(connectors): improve starknet detection on autoconnect

### PR DESCRIPTION
**Pull request by fracek**
This the fix discussed in #69. It's more of a temporary fix while we work on better Argent/Injected detection.

### CODE REVIEW

File: packages/core/src/connectors/injected.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export class InjectedConnector extends Connector<InjectedConnectorOptions> {
   }
 
   available() {
+    getStarknet()
     return globalThis['starknet'] !== undefined
```
Comment (tarrencev) [inline packages/core/src/connectors/injected.ts:24]: I think we could do this:
```suggestion
    return getStarknet() !== undefined
```

**Review by tarrencev — APPROVED** [review]

File: packages/core/src/connectors/injected.ts (hunk 1 above)
Comment (fracek) [reply packages/core/src/connectors/injected.ts:24]: Good point.

Comment (ltoussaint) [reply packages/core/src/connectors/injected.ts]: I tested it with my code and it fails again.

Getting the a pollyfill from `getStarknet` will continue to fail when calling for the `ready` method, it calls `isPreauthorized` which is pollyfilled with a method which throw an Exception.

Comment (tarrencev) [reply packages/core/src/connectors/injected.ts]: Could you share the particular error it is throwing?

**Pull request merged**